### PR TITLE
homepage created successfully

### PR DESCRIPTION
**Pull request by ivinayakg**
Web capture_10-2-2022_04343_127 0 0 1

### CODE REVIEW

File: screens/home/index.html [hunk 1]
```diff
@@ -0,0 +1,126 @@
...
+            <div
+              class="badge m0 t3"
+              icon="false"
+              content="2"
+              color="sec"
+              position="t-right"
+            >
+              <i class="far fa-heart"></i>
+            </div>
+            <div class="header_mobile .text4">Wishlist</div>
```
Comment (HarshnaNHaswani) [inline screens/home/index.html:44]: typo: remove  '.' for text4

File: screens/home/index.html [hunk 2]
```diff
@@ -0,0 +1,126 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Ecom Store</title>
+    <link rel="stylesheet" href="../../css/main.css" />
+    <link rel="stylesheet" href="./index.css" />
```
Comment (HarshnaNHaswani) [inline screens/home/index.html:9]: not using the component library?

File: screens/home/index.html [hunk 3]
```diff
@@ -0,0 +1,126 @@
...
+      </div>
+
+      <div class="header_toggle">
+        <div class="header_burger"></div>
+        <div class="header_burger"></div>
+        <div class="header_burger"></div>
+      </div>
+
+      <div class="header_content">
+        <div class="header_search br-full">
```
Comment (HarshnaNHaswani) [inline screens/home/index.html:25]: `br-full` please keep class name understandable

File: screens/home/index.html [hunk 4]
```diff
@@ -0,0 +1,126 @@
...
+        </div>
+
+        <div class="header_links">
+          <div class="header_link flex">
+            <div
+              class="badge m0 t3"
+              icon="false"
+              content="2"
+              color="sec"
+              position="t-right"
```
Comment (HarshnaNHaswani) [inline screens/home/index.html:40]: isolate styles from html

File: screens/home/index.html [hunk 5]
```diff
@@ -0,0 +1,126 @@
...
+            <div class="catergory_status">New Arrival</div>
+            <div class="catergory_detail">
+              <h3>Catergory Heading</h3>
+              <h4 class="text-light">Catergory Sub Heading</h4>
+            </div>
+          </div>
+        </div>
+        <div class="catergory_card">
+          <div class="catergory_cardImg">
+            <img src="../assets/ecom 4.jpg" alt="" />
```
Comment (HarshnaNHaswani) [inline screens/home/index.html:113]: give alternate source for images

**Review by HarshnaNHaswani — APPROVED** [review]
Looks good to me

File: screens/home/index.html (hunk 1 above)
Comment (ivinayakg) [reply screens/home/index.html:44]: thanks

File: screens/home/index.html (hunk 2 above)
Comment (ivinayakg) [reply screens/home/index.html:9]: using the component library but making some custom changes therefore the component library assets are saved on the root directory
also, there are a lot of custom components I made for this, so this is like a custom library derived from the original component library

File: screens/home/index.html (hunk 4 above)
Comment (ivinayakg) [reply screens/home/index.html:40]: these are custom attributes that will be handled by the JS.

File: screens/home/index.html (hunk 5 above)
Comment (ivinayakg) [reply screens/home/index.html:113]: we just have to put it up there as a static site that's why I'm not doing a total file structuring will do so when will build it as the production-ready app.

**Review by Bibin-Antony — APPROVED** [review]
seems fine

**Pull request merged**